src/components/APIDoc/SchemaDataView.tsx

```diff
@@ -56,15 +61,80 @@ const PropertyView:React.FunctionComponent<PropertyComponentProps> = ({propName,
       </FlexItem>
       <FlexItem>
         <TextContent>
-            <Text component={TextVariants.p}>
-              {propertyType}
-            </Text>
-          </TextContent>
+          <Text component={TextVariants.p}>
+            {propertyType}
+          </Text>
+        </TextContent>
+      </FlexItem>
+      <FlexItem>
+        {extraProps}
       </FlexItem>
     </Flex>
   )
 }
 
+
+interface ExtraPropertyViewProps {
+  propSchema: OpenAPIV3.SchemaObject;
+}
+const ExtraPropertyView:React.FunctionComponent<ExtraPropertyViewProps> = ({propSchema}) => {
+  let maxMin: string | undefined;
+  if (propSchema.maximum && propSchema.minimum) {
+    maxMin = (propSchema.exclusiveMinimum ? '>': '≥') + ` ${propSchema.minimum} and `
+    maxMin += (propSchema.exclusiveMaximum ? '<' : '≤') + ` ${propSchema.maximum}`
+  } else if (propSchema.maximum) {
+    maxMin = (propSchema.exclusiveMaximum ? '<' : '≤') + ` ${propSchema.maximum}`
+  } else if (propSchema.minimum) {
+    maxMin = (propSchema.exclusiveMinimum ? '>' : '≥') + ` ${propSchema.minimum}`
+  }
+
+  let maxMinChar: string | undefined;
+  if (propSchema.maxLength && propSchema.minLength) {
+    maxMinChar = `${propSchema.minLength} to ${propSchema.maxLength} chars`
```

Comment:
The screenshot renders as `7 to 3 chars` and I'm like ... min is 7 and max is 3??

Reply:
Whoops, human error :sweat_smile: